src/main/java/org/littletonrobotics/urcl/URCL.java

```diff
@@ -44,6 +47,10 @@ public class URCL {
   private static RawPublisher periodicPublisher;
   private static RawPublisher aliasesPublisher;
   private static Notifier notifier;
+  private static final DataLog datalog = DataLogManager.getLog();
+  private static RawLogEntry persistentLogEntry = new RawLogEntry(datalog, "URCL/Raw/Persistent");
```

Comment:
The RawLogEntries must be given a type identical to the type used by the NT entries. Otherwise, AdvantageScope will fail to recognize the data as URCL data.

Reply:
Have I set this correctly? AdvantageScope doesn't appear to be recognising it

Reply:
Did you figure out the issue? The types look correct, and AdvantageScope should recognize it in a WPILOG.

Reply:
Nope I can send a log if you want?

Reply:
That would be helpful, yes

Reply:
Nope, AdvantageScope doesn't show anything. We're figuring it out right now.

Reply:
Github wont let me upload the log but its on discord https://discord.com/channels/176186766946992128/528555967827148801/1272012596534448202